Documentation/ceph-csi-drivers.md

```diff
@@ -69,23 +72,18 @@ kubectl create -f cluster/examples/kubernetes/ceph/csi/rbd/snapshot.yaml
 
 ```console
 $ kubectl get volumesnapshotclass
-NAME                      AGE
-csi-rbdplugin-snapclass   4s
+NAME                      DRIVER             DELETIONPOLICY   AGE
+csi-rbdplugin-snapclass   rbd.csi.ceph.com   Delete           3h55m
+
 $ kubectl get volumesnapshot
-NAME               AGE
-rbd-pvc-snapshot   6s
+NAME               READYTOUSE   SOURCEPVC   SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS             SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+rbd-pvc-snapshot   true         rbd-pvc                             1Gi           csi-rbdplugin-snapclass   snapcontent-79090db0-7c66-4b18-bf4a-634772c7cac7   3h50m          3h51m
 ```
 
-In the toolbox pod, run `rbd snap ls [name-of-your-pvc]`.
-The output should be similar to this:
-
-```console
-$ rbd snap ls pvc-c20495c0d5de11e8
-SNAPID NAME                                                                      SIZE TIMESTAMP
-     4 csi-rbd-pvc-c20495c0d5de11e8-snap-4c0b455b-d5fe-11e8-bebb-525400123456 1024 MB Mon Oct 22 13:28:03 2018
-```
+The snapshot will be ready to promote to a new PVC when `READYTOUSE` field of the
+volumesnapshot is set to true
 
-### Restore the snapshot to a new PVC
+### Promote the snapshot to a new PVC
```

Comment:
Does CSI use the term "promote" instead of "restore"? I would have thought "restore" is more common.